os/drivers/lwnl/lwnl.c

```diff
@@ -363,7 +361,7 @@ int lwnl_postmsg(lwnl_cb_status evttype, void *buffer)
 			}
 		}
 	}
-
+	LWNLDEV_UNLOCK(g_lwnl_upper);
```

Comment:
`sem_post` in line 360 sometimes can make context switching. In this case, there is no unlock the `g_lwnl_upper`.
Is it okay?

Reply:
do you have any idea how to fix it?